platooning_control/src/platoon_control_worker.cpp

```diff
@@ -152,6 +152,28 @@ namespace platoon_control
         angVelCmd_ = pp_.getAngularVelocity();
     }
 
+    void PlatoonControlWorker::generateAccel(const double& current_speed, const ros::Time& current_speed_timestamp, const ros::Time& current_time)
+    {
+        const double v_i = current_speed; 
+        const double v_f = speedCmd_; 
+        const double delta_time = (current_time - current_speed_timestamp).toSec();
```

Comment:
are we assuming that current speed and current time are close enough to the speed timestamp and value to use for calculating acceleration?